### PR TITLE
tests: Changed container image source for busybox

### PR DESCRIPTION
**Pull request by olivergs**
Due to docker rate limiting we can not rely in docker.io for retrieving the images.

This was detected when executing our tests for podman fedora gating pipeline. Our busybox image was not downloaded and
one of the list tests was failing.

### CODE REVIEW

**Comment by olivergs** [comment]
recheck

File: images/test/busybox/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+FROM busybox
```
Comment (HarryMichal) [inline images/test/busybox/Dockerfile:1]: Is this file really needed? It is not particularly hard to get a busybox image and there is nothing special about the image (any other image that is not a toolbox image would be sufficient).

**Review by HarryMichal — CHANGES_REQUESTED** [review]
The change looks alright. The reasoning behind switching from Docker to Quay is also fine. Apart from a single comment this looks good to go.

File: images/test/busybox/Dockerfile (hunk 1 above)
Comment (olivergs) [reply images/test/busybox/Dockerfile:1]: I added this file to build the hosted image in the quay.io repository from it. When I created the quay.io image I just uploaded this Dockerfile. We can remove it from our code but I considered to add it to "subliminally" add the information about what image are we using at quay.io and where it comes from.

I know any other image could do the work, but as busybox is the smaller we can find out there, I think busybox is ok for what we need.

Comment (HarryMichal) [reply images/test/busybox/Dockerfile:1]: > I know any other image could do the work, but as busybox is the smaller we can find out there, I think busybox is ok for what we need.

Yes, that is undoubtedly true.

> I added this file to build the hosted image in the quay.io repository from it.

Fair enough. Maybe just add a `README.md` explaining the purpose of the Dockerfile.

One more thing. Maybe rename the file to `Containerfile`. The name is a bit more "generic".

**Review by HarryMichal — APPROVED** [review]
Looks alright. The test failure in Rawhide is unrelated.
